server/jetstream_api.go

```diff
@@ -259,7 +259,22 @@ const JSMaxNameLen = 256
 // TODO(dlc) - Move to more generic location.
 type ApiError struct {
 	Code        int    `json:"code"`
+	Kind        string `json:"kind,omitempty"`
 	Description string `json:"description,omitempty"`
+	URL         string `json:"-"`
+	Help        string `json:"-"`
+}
+
+func (e *ApiError) Error() string {
+	return e.Description + " (" + e.Kind + ")"
+}
+
+func (e *ApiError) CopyF(a ...interface{}) *ApiError {
+	return &ApiError{
+		Code:        e.Code,
+		Kind:        e.Kind,
+		Description: fmt.Sprintf(e.Description, a...),
```

Comment:
We can do the camel case here. Most are fine but if we wrap an err then staticcheck forces those to be all lowercase.